### DISCUSSION TITLE
Where to report websites that cause issues with KeePassXC?

### DISCUSSION THREAD

**Discussion by TESTER-sec** [keepassxreboot/keepassxc#11249] [Q&A] [answered]
There used to be a spot somewhere in this project to report problematic websites for KeePassXC?

After a 30 minute search I cannot find it. Has it been removed? If yes, where are problematic websites to be reported?

**Comment by droidmonkey** [comment, marked as answer]
https://github.com/keepassxreboot/keepassxc-browser/issues/1358

**Reply by TESTER-sec** [reply]
@droidmonkey Thank you.
